data/remnant missions.txt

```diff
@@ -493,5 +598,761 @@ mission "Remnant: Technology Available"
+			`	He shakes his head and continues, "We need to keep our medical facilities as free as possible. Could you take some of our injured to Aventine? They are stable and healthy enough for an unattended transport, and it would free up medics and space here."`
+			choice
+				`	"Of course."`
+					goto transportRS3
+				`	"I'd rather not."`
+					decline
+			label transportRS3
+			`	He looks relieved. "Thanks. I was going to have to divert one of our sentries to take them." He pauses and consults his comlink. "The medical team will have the injured settled onboard shortly, along with everything they need to ensure they arrive safely. They will probably have some fresh crew to send back with you."`
+				accept
+	on stopover
+		event "Remnant Albatross"
+		dialog
+			`The injured crew make for an eerily quite jaunt from Viminal to Aventine. They don't sing, and rarely sign. Some spend the trip mellowed out on powerful painkillers, while the few that leave their rooms seem to be mostly intent on doing prescribed stretches and movements to help damaged muscles heal. A couple are interested in visiting with someone from outside the Ember Wastes, but they lack the energy to do so for long.`
+			`	On Aventine there are ambulances and medical staff waiting to transport them to longer-term medical facilities. As they depart, a fresh group of crew take their places.`
+	on complete
+		payment 100000
+		conversation
+			`The return trip to Viminal is much more interesting. The crew spend more time wandering your ship than they do in their bunks. Some spend most of their time down in engineering, others hang out around the main guns or the cockpit. Only their remarkable aptitude at figuring out where to be in order to not interfere with your crew keeps them from being a hindrance. As you open the hatch they disperse across the tarmac to the ships they have been assigned to, each one of them signing their thanks as they leave.`
+			`	A few minutes after they leave your comlink beeps, notifying you that a message from the prefect has arrived:`
+			`	"Captain <last>! Thank you for your help in defending Viminal and transporting those wounded. I have transfered a small token of our appreciation to you. That being said, perhaps you would appreciate this a bit more: The other prefects and I have agreed to grant you access to a wider range of ships: our Albatross class warships, and Pelican class freighters. I think you would term the former 'heavy warships,' and the later 'heavy freighters.' At the rate these raids are going, it is in everyone's best interests that you have access to a better warship."`
```

Comment:
"transfered" to "transferred", "later" to "latter"

Reply:
Done.